src/components/splash-screen.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { View, Image } from 'react-native';
+import SvgContainer from './SvgContainer';
+
+import SplashScreenStyles from '../styles/splash-screen-styles';
+
+// SVG xml must be converted to a string for react-native-svg
```

Comment:
One workaround is to make a separate react component with the svg which is a lot cleaner and we should store all the svg components in the an `assets` or `icons` folder